ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/423__model_serving_customruntimes.robot

```diff
@@ -54,11 +83,11 @@ Verify RHODS Users Can Deploy A Model Using A Custom Serving Runtime
     Serve Model    project_name=${PRJ_TITLE}    model_name=${model_name}    framework=onnx    existing_data_connection=${TRUE}
     ...    data_connection_name=model-serving-connection    model_path=mnist-8.onnx
     Wait Until Runtime Pod Is Running    server_name=${MODEL_SERVER_NAME}
-    ...    project_title=${PRJ_TITLE}
+    ...    project_title=${PRJ_TITLE}    timeout=15s
     Verify Model Status    ${model_name}    success
     Verify Model Inference    ${model_name}    ${inference_input}    ${exp_inference_output}    token_auth=${TRUE}
     ...    project_title=${PRJ_TITLE}
-
+    
```

Comment:
please remove

Reply:
why? according to the linter, there must be 2 lines btw sections

Reply:
I meant to remove the trailing spaces :slightly_smiling_face: